Sources/Datadog/URLSessionAutoInstrumentation/Interception/URLFiltering/FirstPartyURLsFilter.swift

```diff
@@ -12,9 +12,9 @@ internal struct FirstPartyURLsFilter {
     /// "example.com", "api.example.com", but not "foo.com".
     private let regex: String
 
-    init(configuration: FeaturesConfiguration.URLSessionAutoInstrumentation) {
+    init(firstPartyHosts: Set<String>) {
```

Comment:
Ha, interesting I have a similar change that I haven't pushed yet. I'm going to revert on my end to avoid merge-conflicts :)
Some more compact naming suggestion:
```suggestion
    init(hosts: Set<String>) {
```

Reply:
yes good idea 👍 i'll shorten the name